config/default/kustomization.yaml

```diff
@@ -1,12 +1,12 @@
 # Adds namespace to all resources.
-namespace: etcd-cluster-operator-system
+namespace: eco-system
```

Comment:
I shortened the namespace and the name prefix below because otherwise we end up with service DNS names (and hence certificate common names) which are longer than 64 characters....which cert-manager rejects.

```
Resource: "cert-manager.io/v1alpha2, Resource=certificates", GroupVersionKind: "cert-manager.io/v1alpha2, Kind=Certificate"
Name: "etcd-cluster-operator-serving-cert", Namespace: "etcd-cluster-operator-system"
Object: &{map["apiVersion":"cert-manager.io/v1alpha2" "kind":"Certificate" "metadata":map["annotations":map["kubectl.kubernetes.io/last-applied-configuration":"{\"apiVersion\":\"cert-manager.io/v1alpha2\",\"kind\":\"Certificate\",\"metadata\":{\"annotations\":{},\"name\":\"etcd-cluster-operator-serving-cert\",\"namespace\":\"etcd-cluster-operator-system\"},\"spec\":{\"commonName\":\"$(SERVICE_NAME).$(SERVICE_NAMESPACE).svc\",\"dnsNames\":[\"$(SERVICE_NAME).$(SERVICE_NAMESPACE).svc.cluster.local\"],\"issuerRef\":{\"kind\":\"Issuer\",\"name\":\"selfsigned-issuer\"},\"secretName\":\"webhook-server-cert\"}}\n"] "creationTimestamp":"2019-10-30T12:32:42Z" "generation":'\x01' "name":"etcd-cluster-operator-serving-cert" "namespace":"etcd-cluster-operator-system" "resourceVersion":"15401" "selfLink":"/apis/cert-manager.io/v1alpha2/namespaces/etcd-cluster-operator-system/certificates/etcd-cluster-operator-serving-cert" "uid":"20621af3-a988-4540-875e-1ca8b4b704ab"] "spec":map["commonName":"$(SERVICE_NAME).$(SERVICE_NAMESPACE).svc" "dnsNames":["$(SERVICE_NAME).$(SERVICE_NAMESPACE).svc.cluster.local"] "issuerRef":map["kind":"Issuer" "name":"selfsigned-issuer"] "secretName":"webhook-server-cert"] "status":map["conditions":[map["lastTransitionTime":"2019-10-30T12:32:42Z" "message":"Waiting for CertificateRequest \"etcd-cluster-operator-serving-cert-1917100924\" to complete" "reason":"InProgress" "status":"False" "type":"Ready"]]]]}
for: "STDIN": admission webhook "webhook.cert-manager.io" denied the request: spec.commonName: Too long: must have at most 64 characters

```

Reply:
`ecosystem` nice 😁